src/vs/platform/tunnel/node/tunnelService.ts

```diff
@@ -83,8 +83,10 @@ class NodeRemoteTunnel extends Disposable implements RemoteTunnel {
 	}
 
 	public async waitForReady(): Promise<this> {
+		const startPort = this.suggestedLocalPort ?? this.tunnelRemotePort;
+		const hostname = isMacintosh && startPort < 1024 && isAllInterfaces(this.defaultTunnelHost) && !isPortPrivileged(startPort, this.defaultTunnelHost, OS, os.release()) ? '0.0.0.0' : '127.0.0.1';
```

Comment:
I might be missing something here, but why wouldn't we just do this?

```ts
const hostname = isAllInterfaces(this.defaultTunnelHost) ? '0.0.0.0' : '127.0.0.1';
```

Reply:
ah yeah that will work too, don't really have a preference I just thought it would be still preferred to just use `127.0.0.1` as before for all other cases as it's only a needed due to macos bug